csrc/executor.cpp

```diff
@@ -977,10 +977,16 @@ at::Tensor allocateOutput(
     return alloc_tensor;
   }
 
+  if (alias_info.type == AllocationType::Evaluate && !alias_info.aliased_io) {
+    // Compute non-aliased outputs that were marked for expr evaluator.
+    at::Tensor out_tensor = ee.evaluate(out_tv).as<at::Tensor>();
```

Comment:
```suggestion
    auto out_tensor = ee.evaluate(out_tv).as<at::Tensor>();
```

`as<at::Tensor>` is already clear about the type, so using `auto` as the return type is fine and encouraged.